doc/source/analytics/logging.md

```diff
@@ -80,6 +80,49 @@ The two required environment variables are:
  * LOGGER_KAFKA_BROKER : The Kafka Broker service endpoint.
  * LOGGER_KAFKA_TOPIC : The kafka Topic to log the requests.
 
+### Logging to encrypted Kafka with SSL
+
+You can log requests to an encrypted Kafka with SSL. SSL uses private-key/ certificate pairs, which are used during the SSL handshake process. 
+
+To be able to log payload, the client needs to authenticate with SSL, the client needs its own keystore, made up of a key pair and a signed certificate, signed by the CA certificate file for verifying the broker's certificate. Here is an example below on how to define those for a deployment: 
```

Comment:
:thought_balloon: It might be nice to make this a bit clearer with a list, e.g.
```suggestion
To be able to log payloads, the client needs:
* to authenticate with SSL
* its own keystore, made up of a key pair and a signed certificate
* the CA certificate used to sign the key-certificate pair

The CA certificate needs to be recognised by the broker and can also be used for verifying the broker's certificate.

Here is an example on how to define these for a deployment: 
```

Reply:
:thought_balloon: It would also be quite nice to link to the Confluent Kafka documentation in case users need any further details

Reply:
Do you have [that documentation](https://docs.confluent.io/platform/current/kafka/authentication_ssl.html#clients) in mind or another one?

Reply:
The [overall page](https://docs.confluent.io/platform/current/kafka/authentication_ssl.html) is helpful, but it doesn't really talk about the options we're exposing here, so probably also good to link through to the [librdkafka](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) config properties, as that's how to properly understand what's on offer and what values to provide.

I'll admit it's a bit detailed and implementation-specific, but it's useful.

Reply:
Done! :D